CONTRIBUTING.md

```diff
@@ -51,7 +51,7 @@ Click the “New pull request” button at the top.
 
 Note that FOSSASIA’s repository will be on the left and your repository will be on the right.
 
-Click the green button “Create pull request”. Give a succinct and informative title, in the comment field give a short explanation of the changes and click the green button “Create pull request” again.
+Click the green button “Create pull request”. Give a succinct and informative title. In addition to that in the comment field, give a short explanation of the changes and then click the green button “Create pull request” again.
```

Comment:
If you do want to change it (however I don't think it is not clear), it can read like this instead
```suggestion
Click the green button “Create pull request”. Give a succinct and informative title. In the comment field, give a short explanation of the changes and click the green button “Create pull request” again.
```

`In addition to that` is unnecessarily verbose and `then` before `click` is unecessary which breaks the flow of sentence